modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -0,0 +1,249 @@
+[[uyuni-proxy-setup]]
+= Uyuni Proxy Setup
+
+{productname} Proxy requires additional configuration.
+
+
+
+[[uyuni-proxy-setup-pattern]]
+== Install the [path]``suma_proxy`` pattern
+
+Either, on the server, select the [package]``patterns_uyuni_proxy`` package for installation, or make sure the [path]``uyuni_proxy`` pattern is installed using the following command on the proxy as root:
+
+----
+zypper in -t pattern uyuni_proxy
+----
+
+The new salt-broker service will be automatically started at the end of the package installation.
+This service forwards the Salt interactions to the {productname} server.
+
+// REMARK 2019-08-23, ke: Is this a Salt proxy-only feature?
+[NOTE]
+.Proxy Chains
+====
+It is possible to arrange Salt proxies in a chain.
+In such a case, the upstream proxy is named `parent`.
+====
+
+Make sure the TCP ports `4505` and `4506` are open on the proxy.
+The proxy must be able to reach the {productname} Server or a parent proxy on these ports.
+
+
+
+[[uyuni-proxy-setup-copycert]]
+== Copy Server Certificate and Key
+
+The proxy will share some SSL information with the {productname} Server.
+Copy the certificate and its key from the {productname} 4 Server or the parent proxy.
+
+As root, enter the following commands on the proxy using your {productname} 4 Server or parent Proxy 4 (named [replaceable]``PARENT``):
+
+----
+mkdir -m 700 /root/ssl-build
+cd /root/ssl-build
+scp root@PARENT:/root/ssl-build/RHN-ORG-PRIVATE-SSL-KEY .
+scp root@PARENT:/root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT .
+scp root@PARENT:/root/ssl-build/rhn-ca-openssl.cnf .
+----
+
+
+[NOTE]
+====
+To keep the security chain intact, the {susemgrproxy} functionality requires the SSL certificate to be signed by the same CA as the {productname} Server certificate.
+Using certificates signed by different CAs for proxies and server is not supported.
+====
+
+
+
+[[uyuni-proxy-setup-confproxy]]
+== Run [command]``configure-proxy.sh``
+
+The [command]``configure-proxy.sh`` script will finalize the setup of your {susemgrproxy}.
+
+Now execute the interactive [command]``configure-proxy.sh`` script.
+Pressing kbd:[Enter] without further input will make the script use the default values provided between brackets ``[]``.
+Here is some information about the requested settings:
+
+{productname} Parent::
+A {productname} parent can be either another proxy or a {productname} Server.
+
+HTTP Proxy::
+A HTTP proxy enables your {productname} proxy to access the Web.
+This is needed if direct access to the Web is prohibited by a firewall.
+
+Proxy Version to Activate::
+Normally, the correct value (3.0, 3.1, 3.2, or 4.0) should be offered as a default.
```

Comment:
We should really clean up this, or maybe think about other ways of doing it:
- 3.0 and 3.1 are EoL
- Soon we'll have 4.1 for SUSE Manager
- But at the same time Uyuni is not going to have X.Y versioning but YYYYMM (what value should be added here?)

Reply:
better do not mention the version number here.